Rakefile

```diff
@@ -56,19 +56,17 @@ task :run do
       MessageGenerator.new(github_owner_and_repo).message
     }.compact
 
-    message = if messages.any?
-                "Hello :paw_prints:, this is your <https://github.com/alphagov/govuk-deploy-lag-badger|regular badgering to deploy>!\n\n#{messages.join("\n")}"
-              else
-                "Hello :paw_prints:, there aren't any undeployed pull requests older than 7 days. GOOD JOB TEAM! :#{random_parrot}:"
-              end
-
-    message_payload = {
-      username: "Badger",
-      icon_emoji: ":badger:",
-      text: message,
-      mrkdwn: true,
-      channel: team,
-    }
+    message = "Hello :paw_prints:, this is your <https://github.com/alphagov/govuk-deploy-lag-badger|regular badgering to deploy>!\n\n#{messages.join("\n")}" if messages.any?
```

Comment:
You can now move this string directly into line 65, so that the only conditional check happening is on line 61.

Reply:
Ah yes, too eager to get rid of this that I didn't notice 😄